### PR TITLE
[10] auth_admin_passkey Fix Issue #1227

### PR DESCRIPTION
**Pull request by mohamedhagag**
this is a fix for issue:
https://github.com/OCA/server-tools/issues/1227 
based on the PR by akretion for 8 and 9 here:
https://github.com/OCA/server-tools/pull/519

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please change module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

And check travis status

**Comment by mohamedhagag** [comment]
Version number change will create another PR, is this OK ?

Regarding the travis status I can't find errors related to this module, it complains about other modules like auth_brute_force , auth_user_case_insensitive and other modules.
What should I do ?

Sorry but I'm new in contributing to OSS projects using github.

**Comment by pedrobaeza** [comment]
@mohamedhagag no, adding version number doesn't create other PR if you do it through git, not via GitHub interface.

And the Travis errores are yours:
```
auth_admin_passkey/models/res_config.py:8:1: E302 expected 2 blank lines, found 1
auth_admin_passkey/models/res_config.py:17:63: E231 missing whitespace after ','
auth_admin_passkey/models/res_config.py:32:62: E231 missing whitespace after ','
```

**Comment by mohamedhagag** [comment]
@pedrobaeza would you plz chk and merge.

**Comment by pedrobaeza** [comment]
This is not how it works. Check https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md for the general rules. It requires at least 2 reviews, and a way to get reviews is to make another reviews and ask in return to review yours.

File: auth_admin_passkey/models/res_config.py [hunk 1]
```diff
@@ -4,55 +4,50 @@
...
-            repr(self.auth_admin_passkey_send_to_user))
+    auth_admin_passkey_send_to_admin = fields.Boolean(
+        string='Send email to admin user.',
+        help="""When the administrator use his password to login in """
+             """with a different account, System will send an email """
+             """to the admin user.""")
+
+    auth_admin_passkey_send_to_user = fields.Boolean(
+        string='Send email to user.',
+        help="""When the administrator use his password to login in """
```
Comment (legalsylvain) [inline auth_admin_passkey/models/res_config.py:51]: Hi, 
- you moved a lot of code, that make the diff quite hard to review. I don't see the reason of that ?
- I think adding triple quote is unnecessary.

Other it looks great for me !

**Review by legalsylvain — CHANGES_REQUESTED** [review]

**Comment by NL66278** [comment]
I think the latest two commits, using  the string 'False' instead of an empty string when setting the parameters, and then testing for string 'True' or '1' when getting the parameters is not needed. The second commit fixes an error introduced by the first commit. Setting the parameter to an empty string effectively sets it to a False or rather falsey value. So original code as now existing in 8.0 just should work IMHO.

File: auth_admin_passkey/models/res_config.py (hunk 1 above)
Comment (NL66278) [reply auth_admin_passkey/models/res_config.py:51]: I agree it is a good idea to get rid of the triple quotes. They were introduced in the fix for 8.0.

**Comment by dek-odoo** [comment]
are there any updates ? The issue is still there.

**Comment by gurneyalex** [comment]
superseeded by #1720

**Pull request closed without merging**